### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by teofils1**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Marinescu Teodor <teodor.marinescu03@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/teodor.marinescu/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+| [Male to male jumper wires](https://media.digikey.com/pdf/Data%20Sheets/Digi-Key%20PDFs/Jumper_Wire_Kits.pdf) | Wires | [6.98 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/880-fire-colorate-mama-mama-10p-10-cm.html?search_query=fire+mama+mama&results=63) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+|[embassy-time](https://embassy.dev/book/dev/time_keeping.html)|Time management library  |Used for time-based operations such as delays |
+|[embassy-rp](https://docs.embassy.dev/embassy-rp/git/rp2040/index.html)| Peripheral access library |Used for initializing and interacting with peripherals |
+|[embassy-executor](https://docs.embassy.dev/embassy-executor/git/std/index.html)|Asynchronous executor for Rust embedded systems| Used for task scheduling and asynchronous programming|
+|[gpio](https://docs.embassy.dev/embassy-stm32/git/stm32c011d6/gpio/index.html)|GPIO manipulation |Used for interacting with GPIO pins |
```
Comment (alexandruradovici) [inline docs/project/2024/teodor.marinescu/index.md:84]: This is for STM32, not the Puco.

Comment (teofils1) [reply docs/project/2024/teodor.marinescu/index.md]: https://www.alexdwilson.dev/learning-in-public/gpio-on-the-raspberry-pi-pico
I used this link as documentation.

File: docs/project/2024/teodor.marinescu/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+|[embassy-executor](https://docs.embassy.dev/embassy-executor/git/std/index.html)|Asynchronous executor for Rust embedded systems| Used for task scheduling and asynchronous programming|
+|[gpio](https://www.alexdwilson.dev/learning-in-public/gpio-on-the-raspberry-pi-pico)|GPIO manipulation |Used for interacting with GPIO pins |
+|[pwm](https://docs.embassy.dev/embassy-nrf/git/nrf52840/pwm/index.html)|Pulse-width modulation |Used for controlling the RGB LED light intensity |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [Youtube](https://www.youtube.com/watch?v=G8uTmRrY1M8)
+2. [Youtube](https://www.youtube.com/watch?v=7qFM8cTif8U)
```
Comment (alexandruradovici) [inline docs/project/2024/teodor.marinescu/index.md:92]: Use the names of the videos as titles here.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
@teofils1  is this still in progress?

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**